R/pkg/R/mllib.R

```diff
@@ -53,26 +53,27 @@ setClass("AFTSurvivalRegressionModel", representation(jobj = "jobj"))
 #' @note KMeansModel since 2.0.0
 setClass("KMeansModel", representation(jobj = "jobj"))
 
-#' Saves the machine learning model to the input path
+#' Saves the MLlib model to the input path
```

Comment:
I think the convention that has been suggested is that we have the page title being the same first sentence of the description?